data/mods/Xedra_Evolved/mutations/paraclesians/sylph_spell_learning_eocs.json

```diff
@@ -334,6 +337,24 @@
     ],
     "false_effect": [ { "run_eocs": "EOC_SYLPH_SPELL_EXPERIENCE_INCREASER_SELECTOR" } ]
   },
+  {
+    "type": "effect_on_condition",
```

Comment:
Right after I merged this I had the thought, "Is there a way to set this experience eoc to operate by school instead of by individual spells?"  Or is it locked to that because different spells need different math?"

Reply:
The reason it's all done manually is there's currently no way for EoC to get a list of the spells you know, so they all have to be checked for individually. It can all be radically simplified once that list is accessible (though even then it might need some work to replicate the current "lower Difficulty spells are more likely to be picked than higher-Difficulty ones" behavior).